docs/dashboards/introduction.md

```diff
@@ -24,7 +24,15 @@ In the top right of the screen, click the **+** icon to open the **Create** menu
 
 ![Create Dashboard](images/create.png)
 
-If you don't want to build a dashboard from scratch, or want to experiment by making changes to an existing dashboard without affecting the original, you can **duplicate** an existing dashboard. From an existing dashboard, click on the **...** menu in the upper right, and select **Duplicate**.
+## Duplicating a dashboard
+
+If you don't want to build a dashboard from scratch, or want to experiment by making changes to an existing dashboard without affecting the original, you can **duplicate** an existing dashboard. 
+
+To duplicate a dashboard, click on the **...** menu in the upper right of the dashboard, and select **Duplicate**.
+
+By default, Metabase will create a new dashboard, with copies of the questions in the original dashboard, and save everything to the collection you specify. 
```

Comment:
For another time: does the copy inherit any other properties? Subscriptions, sharing/embedding settings, etc.?

Reply:
It does not get subscriptions. I need to investigate the public sharing/embedding aspect.

Reply:
Looks like it also does not carry over sharing/embedding settings.